qtism/data/storage/xml/XmlCompactDocument.php

```diff
@@ -380,4 +398,9 @@ public function explodeRubricBlocks()
 
         return $references;
     }
+
+    protected function inferVersion()
+    {
+        return '2.1';
```

Comment:
This will also need to be changed to enable compilation to QTI 2.2 tests. Will be part of another PR.